src/lwc/geFormFieldPicklist/geFormFieldPicklist.js

```diff
@@ -57,11 +57,6 @@ export default class GeFormFieldPicklist extends LightningElement {
         return `${this.objectName}.${this.fieldName}`;
     }
 
-    handleValueChange(event) {
```

Comment:
@kenlewis-sfdo - I think we should keep this, stop the event from propagating, and just update the custom event name accordingly.

As it is, the [event](https://git.soma.salesforce.com/aura/lightning-global/blob/6410b243701b052ab8dd9915eb5fec92ba8f2319/ui-lightning-components/src/main/modules/lightning/combobox/combobox.js#L402) (link courtesy of Ben) from the `lightning-combobox` is set to `bubbles: true` and `composed: true` by default. There's a small chance that it'll get unintentionally handled somewhere higher in our hierarchy.